packages/@okta/vuepress-site/docs/guides/saml-inline-hook/main/index.md

```diff
@@ -0,0 +1,338 @@
+
+1. Click **Next** and **Finish**.
+
+The SAML assertion inline hook is now ready for triggering when a user authenticates through the SAML application.
+
+## Preview and test the SAML assertion inline hook
+
+The SAML assertion inline hook is ready for preview and testing. You now have the following applications configured:
+
+* The SAML application (okta-spring-boot-saml-example) is ready to sign in and authenticate users using your Okta org as an IdP.
+* The external service (okta-inlinehook-samlhook) is ready with code to receive and respond to an Okta SAML assertion inline hook call.
+* The Okta org is setup to call the external service when a SAML assertion inline hook is triggered by a user sign-in from the SAML application, and is ready to receive a response.
+
+### Preview the SAML assertion inline hook
+
+1. Navigate to Inline Hooks (**Workflow** > **Inline Hooks**) in your Admin Console.
+
+1. Click on the SAML assertion inline hook name (in this example, "Patient SAML Hook").
+
+1. Click the **Preview** tab.
+
+1. Select a user assigned to your SAML application in the first block titled **Configure Inline Hook request**; that is, a value for the `data.userProfile` field.
+
+1. Select your SAML Application by typing in the **Select a SAML app** field (in this example, "Spring Book SAML"), and clicking on your application.
+
+1. From the **Preview example Inline Hook request** block, click **Generate Request**. You should see the user's request information in JSON format that is sent to the external service.
+
+   > **Note:** You can also **Edit** this call for development or testing purposes.
+
+1. From the **View service's response** block, click **View Response**. You will see the response from your external service in JSON format, which either adds a claim to the SAML assertion or not.
+
+### Test the SAML assertion inline hook
+
+1. Start your SAML Application by navigating to your project folder (`okta-spring-boo-saml-example >`) and then running the application (`> ./gradlew bootRun`).
+
+1. Navigate to your sample application (`http:/localhost:8080`).
+
+1. Navigate to your Glitch.com project to make sure it's listening for requests; open the Console Logs window (**Tools > Logs**).
+
+1. Return to your SAML application and sign in with an Okta user who is NOT in the Patients data store.
+
+      The user should sign in as normal with first name, last name, and email attributes displaying on the sign-in page. In the Glitch log window a message that the user is not part of the data store appears.
+
+1. Sign out of the sample application and now sign in with an Okta user who IS in the Patients data store.
+
+      The user should sign in as normal and also displays a patient ID attribute on the sign-in page, as well as first name, last name, and email address. In the Glitch log window a message displays the patient ID number added to the SAML assertion appears.
```

Comment:
I struggled to parse this one. I hope that I got it right?

```suggestion
      The user should sign in as normal, entering their first name, last name, and email attributes for the fields that display on the sign-in page. The patient ID attribute also appears. In the Glitch log window, a message appears stating that the patient ID number was added to the SAML assertion.
```

Reply:
Essentially, yes. You sign in with your creds and then these credentials are displayed on the "welcome" page -- for lack of a better word -- afterwards, including the patient ID attribute, which is added to the SAML assertion. I might massage the second sentence just to make sure it's understood this attribute appears after signing-in..